### ISSUE TITLE
Rule enforcing handling of subscriptions

### ISSUE THREAD

**Issue by felixfbecker** [cartant/rxjs-tslint-rules#83]
Similar to https://palantir.github.io/tslint/rules/no-floating-promises/, this rule should enforce that the return value of `subscribe()` is handled. This ensures no listeners are leaked.

### Disallowed:
```ts
observable.subscribe()
```

### Allowed:
```ts
subscription.add(observable.subscribe())

this.subscription = observable.subscribe()

return observable.subscribe()
```

**Comment by cartant** [comment]
I've added an `rxjs-no-ignored-subscription` rule to 4.22.0.
